src/i18n/translations/de.json

```diff
@@ -2,23 +2,23 @@
   "attachments": {
     "confirm": {
       "cancel": "Abbrechen",
-      "cancel-title": "Anhang löschen abbrechen",
+      "cancel-title": "Löschen von Anhang abbrechen",
       "delete": "Löschen",
-      "delete-prompt": "Löschen dieses Anhangs entfernt ihn dauerhaft aus deinem Tresor.\nWillst Du den Anhang dauerhaft löschen?",
+      "delete-prompt": "Das Löschen dieses Anhangs wird ihn permanent aus dem Tresor entfernen.\nSind Sie sicher, dass Sie diesen löschen möchten?",
       "delete-prompt-title": "Löschen von \"{{title}}\"",
-      "delete-title": "Anhang löschen bestätigen"
+      "delete-title": "Bestätigung für das Löschen von Anhängen"
     },
     "delete": "Löschen",
     "delete-title": "Anhang löschen",
-    "download": "Herunterladen",
+    "download": "Download",
     "download-title": "Anhang herunterladen",
-    "drop-files": "Datei(en) hierher ziehen, um sie zum Tresor hinzuzufügen",
-    "drop-instruction": "Drag and drop um Anhänge hinzu zufügen"
+    "drop-files": "Um Datei(en) zum Tresor hinzuzufügen, diese hier hin ziehen",
+    "drop-instruction": "Drag & Drop um Anhänge hinzuzufügen"
   },
   "cancel": "Abbrechen",
   "custom-fields": {
-    "change-type": "Ändere Typ:",
-    "delete-field": "Lösche Feld",
+    "change-type": "Typ:",
```

Comment:
the English translation is "Change Type:". The German translation should have the same meaning, something like "Ändere Typ:"

Reply:
Because of this, i didn't do that.
![image](https://user-images.githubusercontent.com/51911097/120035074-4c70ee80-bffe-11eb-9874-b6825ecea2aa.png)

To be fully correct, actually it must be in german "Typ ändern zu:". 
Then it would make sense with the attached other strings for Text and so on...

Reply:
The fully correct English translation of "Typ ändern zu" would be "Change Type to". But I think we should choose a translation that is closer to the meaning of "Change Type". "Change Type" is an action and not an subject like "Änderungstyp".

Reply:
Translating stings 1:1 from english to german isn't always the right way. 
My way is a more logical way to translate this function. 
From my point "Typ ändern:", and this four times in a row, looks strange.

Reply:
if "Typ ändern" looks strange four times in a row, it would also look strange in English, because we have "Change Type" four times in a row. But this is a different discussion. We get the English translation and should translate it as close as possible, unless there is a better interpretation. When you choose "Typ:" it isn't an action anymore, but the dropdown is a list of actions.